packages/CollapsibleCard/src/CollapsibleCard.styles.js

```diff
@@ -31,10 +31,10 @@ export const Label = styled.div`
 `;
 
 export const LabelText = styled.div`
-  ${fontSize(1)}
-  ${lineHeight(-1)}
+  ${fontSize(1)};
   color: ${tokens.color.black};
   font-weight: bold;
+  line-height: 1;
```

Comment:
This is a bit tight.  I feel like we might want `lineHeight(-2)` or `lineHeight(-1)`.  

![Screen Shot 2020-11-09 at 11 02 21 AM](https://user-images.githubusercontent.com/14944896/98584927-5e19c100-227b-11eb-861b-09a5dca087af.png)

Maybe the same for the meta data, but there are not examples of extra long content for that sub component.

cc @sunminsarahlee